utils/text_formatting.tsx

```diff
@@ -343,25 +337,8 @@ export function sanitizeHtml(text: string): string {
 }
 
 // Copied from our fork of commonmark.js
-const emailAlphaNumericChars = '\\p{L}\\p{Nd}';
-const emailSpecialCharacters = "!#$%&'*+\\-\\/=?^_`{|}~";
-const emailRestrictedSpecialCharacters = '\\s(),:;<>@\\[\\]';
-const emailValidCharacters = emailAlphaNumericChars + emailSpecialCharacters;
-const emailValidRestrictedCharacters = emailValidCharacters + emailRestrictedSpecialCharacters;
-const emailStartPattern =
-  '(?:[' +
-  emailValidCharacters +
-  '](?:[' +
-  emailValidCharacters +
-  ']|\\.(?!\\.|@))*|\\"[' +
-  emailValidRestrictedCharacters +
-  '.]+\\")@';
-const reEmail = XRegExp.cache(
-    '(^|[^\\pL\\d])(' +
-    emailStartPattern +
-    '[\\pL\\d.\\-]+[.]\\pL{2,4}(?=$|[^\\p{L}]))',
-    'g',
-);
```

Comment:
I moved to direct regex instead of string, because of so many escaping and with `u` flag it needs even more escaping. So just a regular regex worked

Reply:
https://regex101.com/r/Adx2SD/1